docs/api/spec/components/schemas/ipset.yaml

```diff
@@ -0,0 +1,48 @@
+IpsetNetwork:
+    type: object
+    properties:
+        network:
+            type: string
+            description: The IPv4 address of the network
+
+IpsetRoleId:
+    type: object
+    properties:
+        role_id:
+            type: integer
+            description: The role of the clients node
+
+IpsetIp:
+    type: object
+    properties:
+        ip:
+            type: string
+            description: The IPv4 address of the client
+
+IpsetPort:
+    type: object
+    properties:
+        port:
+            type: string
+            description: The port type:integer example: tcp:65535
+
+IpsetMac:
+    type: object
+    properties:
+        mac:
+            type: string
+            description: The clients MAC address example: 01:23:45:67:89:AB
+
+IpsetType:
+    type: object
+    properties:
+        type:
+            type: string
+            description: The Ipset type values(Reg|Unreg)
```

Comment:
I think you should be able to add an enum attribute here like this:
enum: [Reg, Unreg]